Rasa_Bot/niceday_client/example.py

```diff
@@ -0,0 +1,6 @@
+from niceday_client import NicedayClient
+
+client = NicedayClient()
+
+profile = client.get_profile(38527)
+print(profile)
```

Comment:
Can we instead of this example have a unittest that also serves as an example?

Reply:
I've added a test that is picked up by pytest. However, it requires the niceday-api to be up and running, with a valid (logged in) .env token. Will need to modify the github actions workflow I think.

Reply:
Hmmmm... I think it is a useful test, but maybe we don't want to run it in our CI? Also because at some point the niceday-api will be separate from this repo. 

 Maybe you can add a `@pytest.mark.integration_test` decorator to the test function. Then we can run `pytest -m 'not integration_test'` during CI.

And maybe add a stupid simple unittest without the actual request to niceday-api?

Reply:
Cool, ok. I've created an `integration` pytest mark for this, and also a (toy) unit test with a mocked profile response. It doesn't yet really test anything since mostly the code is uncritically passing on server responses, but once the function gets a bit more detail (e.g. validating the profiles that come back) then we can add to this unit test as needed.